graphql_compiler/compiler/filters.py

```diff
@@ -141,14 +142,24 @@ def _represent_argument(context, argument, inferred_type):
                                           u'not match the inferred required type for this filter: '
                                           u'{} vs {}'.format(tag_inferred_type, inferred_type))
 
+        # See if the argument is colocated with the directive
```

Comment:
nit: Consider expanding on this comment a bit. What does `colocated` mean here, precisely? The reader of the code might be surprised to learn that `colocated` means `part of the same enclosing vertex field` (i.e. `directive_location.at_vertex() == location.at_vertex()`), rather than `on the same property field` (i.e. `directive_location == location`).